pkg/config/config_test.go

```diff
@@ -75,8 +75,6 @@ func TestAcornDNSStates(t *testing.T) {
 			},
 			expectedClusterDomains: []string{".custom.com"},
 			prepare: func(f *mocks.MockReader) {
-				f.EXPECT().Get(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
-				f.EXPECT().List(gomock.Any(), gomock.Any()).Return(nil)
 			},
```

Comment:
nit: seems like the logic below can handle a `nil` `prepare`, so this empty literal can be dropped entirely.

Reply:
shoot. good call. ugh, rushing this PR and it shows. 🤦